sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+# Required metadata
+sonar.projectKey=assertj:core
+sonar.projectName=AssertJ-Core
+sonar.projectVersion=1.0
```

Comment:
should this follow assertj core version ?

Reply:
would that be required to fix the fact that assertj core sonar report are not updated since November 2016 ?
https://sonarcloud.io/dashboard?id=org.assertj%3Aassertj-core

Reply:
I've used this file when running sonar locally and forgot to exclude it from PR. 
About the question on how to re-activate sonarcloud scan, I've done a small investigation and usually this is  done by integrating Travis CI with sonarcloud to automate the process.

For now I will remove that file from this PR and will start another PR to try automation of sonarcloud scans.